catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -224,4 +237,8 @@ public void setFeatureService(FeatureService featureService) {
   public void setEndpointUtil(EndpointUtil util) {
     this.util = util;
   }
+
+  public void setMessageValidatorManager(MessageValidatorManager messageValidatorManager) {
+    this.messageValidatorManager = messageValidatorManager;
+  }
```

Comment:
✏️ Since we have no reason to change the `MessageValidatorManager` at runtime (at least, not for now), I recommend using constructor injection instead. Not a setter. 

Setters are good for data that actually has reason to change and is being supplied by callbacks from the various frameworks we use.